src/app/models/PlanModel.ts

```diff
@@ -0,0 +1,43 @@
+import * as mongodb from 'mongodb';
+import { ClassInterface } from './ClassModel';
+
+const { defaultProfilePicture } = require('../../config/defaultVars.config');
+
+class PlanModel {
```

Comment:
@theodorusandi why does PlanModel not require an interface?

Reply:
Functions and classes don't need interface to work. We use interface to enforce the 'shape' of the particular function/class. We should add the interface for consistency, but I will merge it first since it's not a major thing.